avocado/core/safeloader.py

```diff
@@ -630,9 +630,9 @@ def find_python_tests(module_name, class_name, determine_match, path):
 
             modules_paths = [parent_path,
                              os.path.dirname(module.path)] + sys.path
-            try:
-                _, found_ppath, _ = imp.find_module(parent_module, modules_paths)
-            except ImportError:
+            found_ppath = PathFinder.find_spec(parent_module,
+                                               modules_paths).origin
```

Comment:
same here.

Reply:
Sure, I will maintain the `continue`, but checking for `find_spec` output instead. Thanks!